instrumentation/kamon-spring/src/main/resources/reference.conf

```diff
@@ -0,0 +1,25 @@
+# ================================================== #
+# kamon Spring reference configuration #
+# ================================================== #
+
+kamon.instrumentation.spring {
+   server {
+   }
```

Comment:
This is missing all the settings for the client and server instrumentation. The same as we do for Akka HTTP.